lib/report_formatter/c3.rb

```diff
@@ -87,6 +85,27 @@ def chart_is_stacked?
       %w(StackedBar StackedColumn StackedArea).include?(mri.graph[:type])
     end
 
+    # change structure of chart JSON to performance chart with timeseries data
+    def build_performance_chart_area(maxcols, divider)
```

Comment:
@PanSpagetka : this looks black magic.

Can you, please, create some method that would have a nice name suggesting that you are "chaniging structure..." and add a comment explaining what is changed into what?

+ then call that function from here.

Thx!